src/reporters/base.ts

```diff
@@ -219,6 +219,8 @@ export abstract class BaseReporter {
         price,
         trigger
       )
+      const balanceOf = await this.config.kit.web3.eth.getBalance(this.config.oracleAccount)
+      this.config.metricCollector.oracleBalanceValue(this.config.oracleAccount, Number(balanceOf))
```

Comment:
I guess you decided to put this here because it's when the balance changes, but I think we should move this logic to a separate function and see if there's a better place to call it, preferably outside of this method. Also, the ticket mentions doing this every once in a while, so maybe that's good to add too?